README.md

```diff
@@ -65,6 +65,16 @@ And then execute:
 $ bundle
 ```
 
+## Semantic Versioning
+
+This project conforms to [semver](http://semver.org/). As a result of this policy, you can (and should) specify a dependency on this gem using the [Pessimistic Version Constraint](http://guides.rubygems.org/patterns/) with two digits of precision. For example:
+
+spec.add_dependency 'active_model_serializers', '~> 1.0'
```

Comment:
I would be confused if the example in the readme listed a version that does not exist. I would suggest using the current most recent version. Once we get to `1.0` I am sure a `grep` would catch the outdated version and update it.